docker-compose.yml

```diff
@@ -6,9 +6,13 @@ services:
       dockerfile: Dockerfile
     restart: always
     ports: 
-      - 51773
+
+#      - 51773
+# new Superserver Port
+      - 53773

```

Comment:
actually, SuperServer port again 1972

```
"com.intersystems.ports.default.license-server": "4002",
"com.intersystems.ports.default.superserver": "1972",
"com.intersystems.ports.default.webserver": "52773",
"com.intersystems.ports.default.xdbc": "53773",
```

Reply:
sorry1 something went wrong during typing. the port shoud be - 1972 of course